src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -124,7 +124,7 @@ __vsc_prompt_cmd_original() {
 		unset IFS
 	fi
 	for ((i = 0; i < ${#ADDR[@]}; i++)); do
-		# unset IFS
```

Comment:
Does running `echo $?` as the next command work?

Reply:
that was broken before as you can see in the original issue